lib/Core/ErrorState.cpp

```diff
@@ -132,11 +127,9 @@ void ErrorState::outputErrorBound(llvm::Instruction *inst, double bound) {
     }
   }
 
-  stream << errorVar << " == (";
+  stream << "\nOutput Error: ";
   stream << PrettyExpressionBuilder::construct(e);
-  stream << ") && ";
-  stream << "(" << errorVar << " <= " << bound << ") && ";
-  stream << "(" << errorVar << " >= -" << bound << ")\n";
+  stream << "\nAbsolute bound: " << bound << "\n";
```

Comment:
@Himeshi I think the message is best be "Absolute Bound" with capital "B", as we use capital "E" in "Output Error" as well.